velox/common/memory/Memory.cpp

```diff
@@ -196,7 +174,9 @@ void* MemoryPoolImpl::reallocate(
     free(p, alignedSize);
     auto errorMessage = fmt::format(
         MEM_CAP_EXCEEDED_ERROR_FORMAT,
-        succinctBytes(cap_),
+        // This is not accurate either way. We'll make it the proper memory
```

Comment:
We might just throw an exception in allocator instead of checking this again at the memory pool. Consider cleanup this in followup.